systemtest/src/main/java/io/strimzi/systemtest/kafkaclients/IKafkaClient.java

```diff
@@ -4,11 +4,14 @@
  */
 package io.strimzi.systemtest.kafkaclients;
 
+import java.io.IOException;
+import java.security.KeyStoreException;
+
 public interface IKafkaClient<T> {
 
-    T sendMessages(String topicName, String namespace, String clusterName, int messageCount) throws InterruptedException;
-    T sendMessagesTls(String topicName, String namespace, String clusterName, String kafkaUsername, int messageCount, String securityProtocol) throws InterruptedException;
+    T sendMessages(String topicName, String namespace, String clusterName, int messageCount, int timeout) throws InterruptedException, KeyStoreException, IOException;
```

Comment:
As a general point, with something like a timeout it's worth using the parameter name to convey the units (e.g. `timeoutSeconds` or `timeoutMs`).

Reply:
I understand the idea but this change is on separate PR. Can you also please create a [QE] JIRA for that?